sklearn/ensemble/tests/test_stacking.py

```diff
@@ -561,3 +566,58 @@ def fit(self, X, y):
     msg = "'MyEstimator' object has no attribute 'n_features_in_'"
     with pytest.raises(AttributeError, match=msg):
         stacker.n_features_in_
+
+
+@pytest.mark.parametrize("stack_method", ["auto", "predict"])
+@pytest.mark.parametrize("passthrough", [False, True])
+def test_stacking_classifier_multilabel(stack_method, passthrough):
+    X_train, X_test, y_train, y_test = train_test_split(
+        X_multilabel_r, y_multilabel_r, stratify=y_multilabel_r, random_state=42
+    )
+
+    estimators = [
+        ("knnc", KNeighborsClassifier()),
+        ("dcs", DummyClassifier(strategy="stratified", random_state=42)),
+        ("dcu", DummyClassifier(strategy="uniform", random_state=42)),
+    ]
+    final_estimator = KNeighborsClassifier()
+
+    clf = StackingClassifier(
+        estimators=estimators,
+        final_estimator=final_estimator,
+        passthrough=passthrough,
+        stack_method=stack_method,
+    )
+    clf.fit(X_train, y_train)
+    clf.predict(X_test)
+    clf.predict_proba(X_test)
+    sc = clf.score(X_test, y_test)
+    # (passthrough & stack_method == "predict") => (sc == 0.44)
+    assert not passthrough or stack_method != "predict" or sc == pytest.approx(0.44)
+
+    # (passthrough & stack_method != "predict") => (sc == 0.6)
+    assert not passthrough or stack_method == "predict" or sc == pytest.approx(0.6)
```

Comment:
Such a check will be flaky. We should not test against a specific performance metric here.

I assume that checking the shape of the prediction, the attribute `classes_` for the classifier and the shape of the `transform` method would be a good start.

Reply:
It should then be parametrized with `passthrough=True/False` as well.